libp2p/CHANGELOG.md

```diff
@@ -158,90 +158,26 @@
 
 - Bump MSRV to 1.65.0.
 
-- Update individual crates.
-    - Update to [`libp2p-dcutr` `v0.9.0`](protocols/dcutr/CHANGELOG.md#090).
-
-    - Update to [`libp2p-ping` `v0.42.0`](protocols/ping/CHANGELOG.md#0420).
-
-    - Update to [`libp2p-request-response` `v0.24.0`](protocols/request-response/CHANGELOG.md#0240).
-
-    - Update to [`libp2p-kad` `v0.43.0`](protocols/kad/CHANGELOG.md#0430).
-
-    - Update to [`libp2p-floodsub` `v0.42.0`](protocols/floodsub/CHANGELOG.md#0420).
-
-    - Update to [`libp2p-autonat` `v0.10.0`](protocols/autonat/CHANGELOG.md#0100).
-
-    - Update to [`libp2p-relay` `v0.15.0`](protocols/relay/CHANGELOG.md#0150).
-
-    - Update to [`libp2p-identify` `v0.42.0`](protocols/identify/CHANGELOG.md#0420).
-
-    - Update to [`libp2p-rendezvous` `v0.12.0`](protocols/rendezvous/CHANGELOG.md#0120).
-
-    - Update to [`libp2p-metrics` `v0.12.0`](misc/metrics/CHANGELOG.md#0120).
-
-    - Update to [`libp2p-swarm` `v0.42.0`](swarm/CHANGELOG.md#0420).
-
-    - Update to [`libp2p-mdns` `v0.43.0`](protocols/mdns/CHANGELOG.md#0430).
-
-    - Update to [`libp2p-gossipsub` `v0.44.0`](protocols/gossipsub/CHANGELOG.md#0440).
-
-    - Update to [`libp2p-yamux` `v0.43.0`](muxers/yamux/CHANGELOG.md#0430).
-
-    - Update to [`libp2p-mplex` `v0.39.0`](muxers/mplex/CHANGELOG.md#0390).
-
-    - Update to [`libp2p-wasm-ext` `v0.39.0`](transports/wasm-ext/CHANGELOG.md#0390).
```

Comment:
I deleted all of these because we cannot link to packages that have been removed, in particular `libp2p-wasm-ext` here. Not sure if that is the best approach. I didn't want to spend much time on old changelog entries so I just went for deleting these parts.